docker_utils/rtenvsub.sh

```diff
@@ -20,10 +20,82 @@ version='0.1'
 # Set a safe umask
 umask 0077
 
-# Import the standard shell library
-# shellcheck source=../shtdlib.sh disable=SC1091
-source "$(dirname "${0}")/../shtdlib.sh" &> /dev/null || ../shtdlib.sh &> /dev/null || source ./shtdlib.sh &> /dev/null || source shtdlib.sh
+default_library_name='shtdlib.sh'
+default_base_download_url='https://raw.githubusercontent.com/sdelements/shtdlib/master'
+default_install_path='/usr/local/bin'
+
+# Temporary debug function
+type -t import | grep -q '^function$' || function debug { echo "${@:2}" ; }
+
+# Import or source
+function import_or_source {
+    if type -t import | grep -q '^function$' ; then
+        debug 10 "Importing ${0}"
+        import "${1}"
+    else
+        debug 10 "Sourcing ${0}"
```

Comment:
Could we print the full path of the library getting imported? It may not be clear which library is getting sourced if you have multiple copies of shtdlib.sh in different locations.

Reply:
That's actually fairly complicated at this point since we don't really know which library will get imported, that depends on the current path and the PATH variable of the shell executing the command and the shell itself.

The only reliable way would be to place something in the script that's being imported and print from there, otherwise we're just making an educated guess at what the shell will do.